### PR TITLE
Add authorization via cancancan gem

### PR DESCRIPTION
**Pull request by MaxCodeX**

### CODE REVIEW

**Comment by truongkma** [comment]
ok

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.for(:account_update) {|u| u.permit(:name, :chat_word, :email,
       :password, :password_confirmation, :current_password)}
   end
+
+  rescue_from CanCan::AccessDenied do |exception|
+    redirect_to root_url, :alert => exception.message
```
Comment (hellovietnam93) [inline app/controllers/application_controller.rb]: convention

File: app/models/ability.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
```
Comment (hellovietnam93) [inline app/models/ability.rb]: bỏ ()

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**